applicationset/examples/git-generator-directory/excludes/cluster-addons/exclude-helm-guestbook/values.yaml

```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: gcr.io/heptio-images/ks-guestbook-demo
+  repository: quay.io/argoprojlabs/argocd-e2e-container
```

Comment:
Here the `argocd-e2e-container` image is used without a tag

Reply:
the tag field is mentioned below. Please see [this](https://github.com/argoproj/argo-cd/pull/23058/files/02e450177c5fc80f2c6c46bfec507466f4445c32#diff-5dec2cf6e00f2b4615502e92b3dda5327ebcc85c3ffa426edae3e828ea2c598cR9).